.buildkite/pipelines/artifacts.yml

```diff
@@ -73,13 +73,18 @@ steps:
 
   - command: .buildkite/scripts/steps/artifacts/cloud.sh
     label: 'Cloud Deployment'
-    soft_fail: true
+    soft_fail:
+      - exit_status: 255
     agents:
       queue: n2-2
     timeout_in_minutes: 30
     if: "build.env('RELEASE_BUILD') == null || build.env('RELEASE_BUILD') == '' || build.env('RELEASE_BUILD') == 'false'"
     retry:
       automatic:
+        # Matches buildkite forced agent shutdown (timeout_in_minutes) and ecctl create failures
```

Comment:
Note that you'll only get a `255` if the job times out AND the job gracefully stops. If the job has to be hard-killed because it's taking too long to exit after the timeout, you'll get a `-1`.

Reply:
Hmm, -1 could also be a pre-emption hard-kill too right?

Reply:
Yep, anything where the job or agent shuts down not gracefully. Problem with the GCP instance, OOM kill, etc

Reply:
pushed https://github.com/elastic/kibana/pull/137775/commits/b774d164baebab3e67652d6f93018d85689452a3